src/components/ui/Block.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import classNames from 'classnames';
+
+const Block = ({ icon, title, children, className }) =>
```

Comment:
Not a bit deal, but I'm not sure this component should be stored in this folder, as it's only for POI block for now. Having a name as generic as "Block" can be misleading.

Reply:
I moved it to `src/poi/blocks/Block.jsx`